### PR TITLE
[dashboard] Always use Public API for workspace stop & delete

### PR DESCRIPTION
**Pull request by easyCZ**
## Related Issue(s)

Fixes #

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - mp-dashboa62efb578df</li>
	<li><b>🔗 URL</b> - <a href="https://mp-dashboa62efb578df.preview.gitpod-dev.com/workspaces" target="_blank">mp-dashboa62efb578df.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/data/workspaces/delete-inactive-workspaces-mutation.ts [hunk 1]
```diff
@@ -6,17 +6,14 @@
...
 import { getListWorkspacesQueryKey, ListWorkspacesQueryResult } from "./list-workspaces-query";
 import { useCurrentOrg } from "../organizations/orgs-query";
-import { useFeatureFlag } from "../featureflag-query";
 
 type DeleteInactiveWorkspacesArgs = {
     workspaceIds: string[];
 };
 export const useDeleteInactiveWorkspacesMutation = () => {
     const queryClient = useQueryClient();
-    const usePublicApiWorkspacesService = useFeatureFlag("publicApiExperimentalWorkspaceService");
```
Comment (geropl) [inline components/dashboard/src/data/workspaces/delete-inactive-workspaces-mutation.ts:19]: Random side note: This feature flag is off for Dedicated. Intentional?

**Comment by geropl** [comment]
Testing...

File: components/dashboard/src/data/workspaces/delete-inactive-workspaces-mutation.ts (hunk 1 above)
Comment (easyCZ) [reply components/dashboard/src/data/workspaces/delete-inactive-workspaces-mutation.ts:19]: Probably not

**Review by geropl — APPROVED** [review]
Code LGTM, tested and works :heavy_check_mark:

**Pull request merged**